src/app/shell/provider/provider-config/provider-config.component.scss

```diff
@@ -1,21 +1,387 @@
-form{
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  -webkit-appearance: none;
+  margin: 0;
+}
+
+input[type=number] {
+  -moz-appearance: textfield;
+}
+
+
+@mixin form-style {
+  width: 410px;
+  height: 34px;
+  padding: 8px 14px 8px 16px;
+  font-size: 13px;
+  line-height: 18px;
+  border: 1px solid #E3E3E3;
+  box-sizing: border-box;
```

Comment:
what the reason to set border box in star selector and in specific selectors too?